lib/pavilion/pav_test.py

```diff
@@ -515,7 +519,8 @@ def _build(self, build_dir):
                                         stdout=build_log,
                                         stderr=subprocess.STDOUT)
 
-                timeout = self.BUILD_SILENT_TIMEOUT
+                timeout = BUILD_SILENT_TIMEOUT if BUILD_SILENT_TIMEOUT is not \
```

Comment:
1. If you have to wrap it, it's too long for an inline if.
2. If there is not timeout, why set one? None is a valid argument for proc.wait.

Reply:
I swear I tried to allow for it, but got an error.  I can try again.  It may have been something else.